test/e2e/image-upload.e2e.ts

```diff
@@ -234,12 +232,6 @@ test.describe('Image upload', () => {
 
       await page.click('role=button[name="Upload image"]')
 
-      const steps = page.locator('div[data-status]')
-
-      for (const step of await steps.all()) {
-        await expect(step).toHaveAttribute('data-status', 'ready')
-      }
-
```

Comment:
When I ran this locally, it passed because `steps.all()` was coming back empty, so the whole loop is being skipped. Boo. This is due to this approach being sort of a hack — normal `expect` will [auto-wait](https://playwright.dev/docs/actionability) a bit for the thing to become visible before asserting, but this `all()` approach just gets the list of things matching at the moment it's called, and if it's empty, it's empty.